doc/classes/ArrayOccluder3D.xml

```diff
@@ -15,13 +15,18 @@
 			<param index="0" name="vertices" type="PackedVector3Array" />
 			<param index="1" name="indices" type="PackedInt32Array" />
 			<description>
+				Sets [member indices] and [member vertices], while updating the final occluder only once after both values are set.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="indices" type="PackedInt32Array" setter="set_indices" getter="get_indices" default="PackedInt32Array()">
+			The occluder's index position. Indices determine which points from the [member vertices] array should be drawn, and in which order.
```

Comment:
Not sure if this part is 100% correct: is it possible for the `indices` array's size to differ from `vertices`?

Reply:
I think the indices array does differ in size. Indices usually repeat while vertices usually don't.